src/networkd.c

```diff
@@ -984,6 +996,19 @@ _netplan_netdef_write_network_file(
         g_string_append_printf(network, "\n[IPoIB]\nMode=%s\n", netplan_infiniband_mode_name(def->ib_mode));
     }
 
+    /* ipv6-ra-overrides */
+    if (ipv6_ra_overrides_is_dirty(&def->ipv6_ra_overrides)) {
+        g_string_append(network, "\n[IPv6AcceptRA]\n");
+        
+        /* Only write RA options that differ from the networkd default. */
+        if (!def->ipv6_ra_overrides.use_dns)
```

Comment:
We also should write this option if it's true, even if `true` is the default networkd value. As I suggested above, by defining it as a tristate, you can write it as true or false and completely omit it if the state is unset.